source/camera/FastPODAllocator.h

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright (C) 2011-2014  OpenDungeons Team
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef GMS_FASTPODALLOCATOR_H_
+#define GMS_FASTPODALLOCATOR_H_
+
+#include <boost/pool/pool.hpp>
+#include <cassert>
+
+namespace gms
+{
+
+/// Make the allocation/deallocation faster for small NON-POLYMORPHIC objects.
+///
+/// Usage:
+///
+/// class A { };
+/// =>
+/// class A : public gms::FastPODAllocator<A> { };
+///
+/// Note that FastPODAllocator<> is not thread safe.
+/// Use as you wish.
+/// @see http://www.boost.org/
+/// @author Hannu Kankaanp�� (hkankaan@cc.hut.fi)
+template <class T>
+class FastPODAllocator
```

Comment:
This should be excluded, it is used with other , non - present mechanism here of creature culling when using QuadTrees.